Error in examples/node/main.ts: Cannot find module '@blyss/sdk/node'

Hi,

I tried to run the Node example at `sdk/examples/node` but got this error.

I'm on git commit `aea24b915606bbba12fe80a9c99ee141b01f3eb2` and I'm running Node v18.14.0.

```
$ npx ts-node main.ts
Error: Cannot find module '@blyss/sdk/node'
Require stack:
- /home/user/blyss/sdk/examples/node/main.ts
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1047:15)
    at Function.Module._resolveFilename.sharedData.moduleResolveFilenameHook.installedValue [as _resolveFilename] (/home/user/blyss/sdk/examples/node/node_modules/@cspotcode/source-map-support/source-map-support.js:811:30)
    at Function.Module._load (node:internal/modules/cjs/loader:893:27)
    at Module.require (node:internal/modules/cjs/loader:1113:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at Object.<anonymous> (/home/user/blyss/sdk/examples/node/main.ts:2:15)
    at Module._compile (node:internal/modules/cjs/loader:1226:14)
    at Module.m._compile (/home/user/blyss/sdk/examples/node/node_modules/ts-node/src/index.ts:1618:23)
    at Module._extensions..js (node:internal/modules/cjs/loader:1280:10)
    at Object.require.extensions.<computed> [as .ts] (/home/user/blyss/sdk/examples/node/node_modules/ts-node/src/index.ts:1621:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/home/user/blyss/sdk/examples/node/main.ts' ]
}
```

--- a/.github/workflows/build-js.yml
+++ b/.github/workflows/build-js.yml
@@ -17,7 +17,7 @@ jobs:
       - name: Use Node.js
         uses: actions/setup-node@v3
         with:
-          node-version: '19.x'
+          node-version: 'lts/*'
 
       - name: Install wasm-pack
         uses: jetli/wasm-pack-action@f98777369a49686b132a9e8f0fdd59837bf3c3fd

--- a/examples/node-local/package.json
+++ b/examples/node-local/package.json
@@ -1,6 +1,6 @@
 {
   "dependencies": {
-    "@blyss/sdk": "0.1.7-alpha.1"
+    "@blyss/sdk": "latest"
   },
   "devDependencies": {
     "ts-node": "^10.9.1",

--- a/examples/node/main.ts
+++ b/examples/node/main.ts
@@ -6,7 +6,7 @@ async function main() {
   const client: Client = new blyss.Client('<YOUR API KEY HERE>');
 
   // Create the bucket
-  const bucketName = 'global.wc-v1';
+  const bucketName = 'state-capitals';
   if (!(await client.exists(bucketName))) {
     console.log('creating...');
     await client.create(bucketName, true, {

--- a/js/tests/bloom.test.ts
+++ b/js/tests/bloom.test.ts
@@ -1,5 +1,9 @@
 import { bloomLookup, bloomWrite, bloomInit } from '../data/bloom';
 
+if (typeof crypto === 'undefined')
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  (globalThis as any).crypto = (require('node:crypto') as any).webcrypto;
+
 describe('bloom filter write + lookup', () => {
   it.each([
     [10, 24, ['a', 'b', 'c']],

--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "author": "Samir Menon <samir@blyss.dev>",
   "name": "@blyss/sdk",
-  "version": "0.1.7-alpha.1",
+  "version": "0.1.7",
   "description": "Blyss SDK, enabling private retrievals from Blyss buckets",
   "type": "module",
   "main": "./dist/index.js",

